components/ViewArticleListPage/index.tsx

```diff
@@ -0,0 +1,219 @@
+        marginBottom="25px"
+      >
+        <AppBar position="static" color="transparent" elevation={0}>
+          <Tabs
+            value={value}
+            onChange={handleChange}
+            indicatorColor="primary"
+            textColor="primary"
+            variant="fullWidth"
+            aria-label="action tabs example"
+          >
+            <Tab label="All" icon={<PersonIcon />} iconPosition="start" />
+            <Tab label="Draft" icon={<DraftsIcon />} iconPosition="start" />
+            <Tab
+              label="Published"
+              icon={<UploadFileIcon />}
+              iconPosition="start"
+            />
+            <Tab label="Archived" icon={<LockIcon />} iconPosition="start" />
+          </Tabs>
+        </AppBar>
+      </Box>
+
+      <Box sx={{ width: '100%' }}>
+        <Paper sx={{ width: '100%', mb: 2 }}>
+          <TableContainer>
+            <Table
+              sx={{ minWidth: 750 }}
+              aria-labelledby="tableTitle"
+              size="medium"
+            >
+              <ArticleTableHead
+                order={order}
+                orderBy={orderBy}
+                onRequestSort={handleRequestSort}
+              />
+              <TableBody>
+                {/* if you don't need to support IE11, you can replace the `stableSort` call with:
+              rows.sort(getComparator(order, orderBy)).slice() */}
+                {stableSort(rows, getComparator(order, orderBy))
+                  .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                  .map((row, index) => {
+                    const labelId = `enhanced-table-checkbox-${index}`
+
+                    return (
+                      <TableRow
+                        hover
+                        role="row"
+                        tabIndex={-1}
+                        key={row.name}
+                        sx={{
+                          '&:hover': {
+                            backgroundColor: '#1976D214 !important',
+                          },
+                        }}
+                      >
+                        <TableCell
+                          component="th"
+                          id={labelId}
+                          scope="row"
+                          sx={{
+                            // Check how to calculate this dyanmically
+                            maxWidth: '150px',
+                          }}
+                        >
+                          <Typography noWrap>{row.name}</Typography>
+                        </TableCell>
+                        <TableCell align="left">{row.date_created}</TableCell>
+                        <TableCell align="left">{row.last_modified}</TableCell>
+                        <TableCell align="left">{row.status}</TableCell>
+                        <TableCell
+                          align="left"
+                          onClick={() =>
+                            router.push('/content/articles/dsadsa')
```

Comment:
understand that this is now draft but think u can already push it to /content/articles/<articlename_id> already